resources/options_lookup.tsv

```diff
@@ -1756,23 +1756,23 @@ Clothes Dryer	None	ResidentialApplianceClothesDryer	fuel_type=electric	cef=2.70
 		ResidentialAirflow	clothes_dryer_exhaust=0																																												
 Lighting	None																																														
 Lighting	100% Incandescent	ResidentialLightingInterior	hw_cfl=0	hw_led=0	hw_lfl=0	pg_cfl=0	pg_led=0	pg_lfl=0	in_eff=15	cfl_eff=55	led_eff=80	lfl_eff=88	option_type=Lamp Fractions	energy_use_interior=900	sch_option_type=Calculated Lighting Schedule	"monthly_sch=1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248"																															
-		ResidentialLightingOther	hw_cfl=0	hw_led=0	hw_lfl=0	pg_cfl=0	pg_led=0	pg_lfl=0	in_eff=15	cfl_eff=55	led_eff=80	lfl_eff=88	option_type=Lamp Fractions	energy_use_garage=100	energy_use_exterior=300	sch_option_type=Calculated Lighting Schedule	"weekday_sch=0.04, 0.037, 0.037, 0.036, 0.033, 0.036, 0.043, 0.047, 0.034, 0.023, 0.024, 0.025, 0.024, 0.028, 0.031, 0.032, 0.039, 0.053, 0.063, 0.067, 0.071, 0.069, 0.059, 0.05"	"weekend_sch=0.04, 0.037, 0.037, 0.036, 0.033, 0.036, 0.043, 0.047, 0.034, 0.023, 0.024, 0.025, 0.024, 0.028, 0.031, 0.032, 0.039, 0.053, 0.063, 0.067, 0.071, 0.069, 0.059, 0.05"	"monthly_sch=1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248"																												
+		ResidentialLightingOther	hw_cfl=0	hw_led=0	hw_lfl=0	pg_cfl=0	pg_led=0	pg_lfl=0	in_eff=15	cfl_eff=55	led_eff=80	lfl_eff=88	option_type=Lamp Fractions	energy_use_garage=100	energy_use_exterior=300	sch_option_type=Calculated Lighting Schedule	"weekday_sch=0.046, 0.046, 0.045, 0.045, 0.046, 0.045, 0.044, 0.041, 0.036, 0.03, 0.024, 0.016, 0.012, 0.011, 0.011, 0.012, 0.019, 0.038, 0.048, 0.06, 0.083, 0.098, 0.085, 0.059"	"weekend_sch=0.046, 0.046, 0.045, 0.045, 0.046, 0.045, 0.044, 0.041, 0.036, 0.03, 0.024, 0.016, 0.012, 0.011, 0.011, 0.012, 0.019, 0.038, 0.048, 0.06, 0.083, 0.098, 0.085, 0.059"	"monthly_sch=1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248"																												

```

Comment:
@mdahlhausen For this schedule to actually get used, you need to change `sch_option_type` to "User-Specified Lighting Schedule"

Reply:
Good catch. I've updated to use the User-Specified Lighting Schedule.  At some point in the future someone can make a calculated version for exterior lights.